backend/apps/notifications/views.py

```
@@ -192,3 +215,128 @@ def post(self, request):
             recipient=request.user, is_read=False
         ).update(is_read=True)
         return Response({"marked_read": updated}, status=status.HTTP_200_OK)
+
+
+class TrackOpenView(APIView):
+    """GET /api/notifications/track/open/<delivery_id>/"""
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)